avocado/core/loader.py

```diff
@@ -44,6 +44,41 @@
 ALL = True
 
 
+def filter_test_tags(test_suite, filter_by_tags):
+    '''
+    The filtering mechanism is limited to instrumented tests with tags set
+    '''
```

Comment:
Sure, I can comment with `"""` but only if all classes use `'''`. If we mix them I can't comment multiple blocks with different style. So as usual, it's not a strong opinion, just an ask for favor.